packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.types.ts

```diff
@@ -43,9 +43,18 @@ export interface IDetailsRowFieldsProps extends IOverrideColumnRenderProps {
   shimmer?: boolean;
 
   /**
-   * Required prop to be passed in from the parent DetailsRow a map of classNames and its mergestyle-created classNames
+   * Subset of classnames currently generated in DetailsRow that are used within DetailsRowFields.
    */
-  rowClassNames: { [className in keyof IDetailsRowStyles]: string };
+  rowClassNames: {
```

Comment:
`DetailsRowFields` is not publicly exported and I'm modifying this signature so that we only accept the classNames that `DetailsRowFields` applies. The remainder from `IDetailsRowStyles` are unnecessary in the context of `DetailsRowFields`.